README.md

```diff
@@ -44,54 +62,126 @@ This goes both ways: when explaining something make sure to present enough conte
+
+We push changes multiple times a day to (remote) branches. We always push changes at the end of the day.
 
 A teammate must review your PR, get their questions answered, and give a +1 as approval before you can merge your changes. The main branch is [protected](https://help.github.com/articles/about-required-reviews-for-pull-requests/). The reason we review code is not just to catch bugs, but because we want to improve the quality of code that we ship. We want our code to be maintainable, easily understood, and [stylish](https://github.com/DoSomething/code-style). If you push code to production, you’re implicitly vouching for it as an example of how to write code at DoSomething. When possible, we use StyleCI to enforce coding style and conventions. StyleCI gets called for each PR and update.
 
-#### Making a pull request
+#### Making a Pull Request
 
-* If your code is not ready for review, or is a prototype that won't be merged, don't create a pull request—or create one prefixed with DON’T MERGE in the title
-* When you create a pull request, fill out the [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates)
-* Highlight any areas you would particularly like review on
+* If your code is not ready for a full review, create a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/). These are great for getting initial feedback on a direction you're taking. If it's a prototype that won't be merged, create a pull request prefixed with DON’T MERGE in the title. 
+* When you create a pull request, fill out the [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates).
+* Highlight any areas you would particularly like review on.
 
-#### Giving feedback on a pull request
+#### Giving Feedback on a Pull Request
 
-* Ask questions, don't make demands
+* Ask questions, don't make demands.
 * Be mindful about how comments come off: "Did you think about" is better than "Why didn't you just..."
-* Compliment! If you see something you like, let them know
-* If you see multiple examples of the same thing \(spelling, caps\) don't comment every time, rather make one comment saying you noticed it throughout
-* Give context for your comments, so they don’t read as arbitrary
-* If you see a code smell, point them to the [prescription](https://sourcemaking.com/refactoring/smells)
+* Compliment! If you see something you like, let them know.
+* If you see multiple examples of the same thing (spelling, caps) don't comment every time, rather make one comment saying you noticed it throughout.
+* Give full context for your comments, so they don’t read as arbitrary.
+* If you see a code smell, point them to the [prescription](https://sourcemaking.com/refactoring/smells).
+* Use PRs as a learning opportunity - ask questions to learn more even if you don't have specific feedback on the work itself.
```

Comment:
This is particularly interesting to me -- wonder if there's a way we could/should assess these types of questions to inform future dev RTs? Are there trends or are they pretty specific and one off?

Reply:
That's a good point! I'm going to add:

If you see that there's confusion / back and forth on an issue/set of issues, that could be an indication that a Dev Roundtable on the topic should be scheduled - propose the topic so that it can be scheduled and organized!